repos/system_upgrade/common/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -418,17 +418,20 @@ def gather_target_repositories(context, indata):
     if missing_custom_repoids:
         raise StopActorExecutionError(
             message='Some required custom target repositories are not available.',
-            details={'hint': (
-                ' The most probably you are using custom or third party actor'
-                ' that produces CustomTargetRepository message or you did a typo'
-                ' in one of repoids specified on command line for the leapp --enablerepo'
-                ' option.'
-                ' Inside the upgrade container, we are not able to find such'
-                ' repository inside any repository file. Consider use of the'
-                ' custom repository file regarding the official upgrade'
-                ' documentation or check whether you did not do a typo in any'
-                ' repoids you specified for the --enablerepo option of leapp.'
-                )
+            details={
+                'hint': (
+                    ' The most probably you are using custom or third party actor'
```

Comment:
'The most probably' -> 'It is most likely that'

Although all this text could be better served by changing the tone to remove all the 'you' and 'we' out of it, because that's not very nice. :)